### DISCUSSION TITLE
vercel deploy

### DISCUSSION THREAD

**Discussion by codewizard0803** [amondnet/vercel-action#228] [Q&A] [answered]
When I deploy project which uses Next.js + TypeScript on Vercel, I has bcrypt module error.
please let me know about this issue.

**Comment by Topdev83** [comment, marked as answer]
I think it's because of package-lock.json or bcrypt module version.
You need to reinstall  @types/bcryptjs or delete package-lock.json file and re-deploy on vercel.
